src/common/args.h

```diff
@@ -152,12 +224,71 @@ class ArgsManager
     bool UseDefaultSection(const std::string& arg) const EXCLUSIVE_LOCKS_REQUIRED(cs_args);
 
  public:
+    ArgsManager();
+    ~ArgsManager();
+
+    /**
```

Comment:
This grouping isn't fully working due to the lack of a group name: https://www.doxygen.nl/manual/grouping.html#memgroup